documentation/source/user_section/installation/windows.rst

```diff
@@ -4,60 +4,55 @@
 Installation for Windows
 ************************
 
-Requirements
-============
-
-Supported Operating Systems
----------------------------
-
-* Windows 10 with Windows Subsystem for Linux (WSL)
-
-
-Installation Options
-====================
+We provides two different ways to install SCT on Windows machines. You can either use Windows Subsystem for Linux (WSL), or Docker.
 
 Option 1: Install on Windows 10 with WSL
 ----------------------------------------
 
+Basic installation (No GUI)
+***************************
+
 Windows Subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs, such as SCT.
 
 #. Install Windows Subsystem for Linux (WSL)
 
-   - Install `Xming <https://sourceforge.net/projects/xming/>`_.
+   - Follow the instructions on Microsoft's `Install WSL <https://docs.microsoft.com/en-us/windows/wsl/install>`_ page.
+
+#. Decide between WSL Version 1 or WSL Version 2.
+
+   By default, Microsoft's instructions will create an Ubuntu environment using Version 2 of WSL. While version 2 has been tested to work with SCT, our development team tests more thoroughly using Version 1 of WSL (due to `better support from GitHub Actions <https://github.com/actions/virtual-environments/issues/50>`_).
+
+   Because of this, your best guarantee for a stable installation of SCT is to convert the WSL2 Ubuntu environment to use WSL1 before proceeding. To do this, first close Ubuntu, then run the following command in a Windows command prompt:
```

Comment:
"your best guarantee for a stable installation of SCT is to convert the WSL2 Ubuntu environment to use WSL1 before proceeding."

i find this sentence difficult to understand (as an ESL 😅 ), and the person I guided through the installation also find it confusing (ie: does it mean that we should use WSL2 or WSL1?)

Reply:
Thanks for the feedback! I'll try to simplify the language.